store.html

```diff
@@ -0,0 +1,191 @@
+        </div>
+    </fieldset>
+
+        <fieldset class="city-postcode"> 
+        <div class="city">
+            <label for="city">City <span class="requiredstar">*</span></label>
+            <select name="city" id="city" required>
+            
+            <option value="" >Select your city... </option>
+            <option value="brighton">Brighton</option> 
+            <option value="cambridge">Cambridge</option>
+            <option value="oxford">Oxford</option>
+            <option value="bristol">Bristol</option>
+            <option value="liverpool"> Liverpool</option>
+            <option value="manchester"> Manchester</option>
+            <option value="birmingham">Birmingham</option>
+            <option value="bournemouth">Bournemouth</option>
+         </select>
+        </div>
+        <div class="postcode">
+            <label for="postcode">Postcode<span class="requiredstar">*</span></label>
+            <input name="postcode" id="postcode" required>
+        </div>
+        </fieldset>
+<!--colour-->
+            
+                <div class="karmaorange">
+
+                    <div class="select-container">
+
+                    
+             <fieldset class="colour">
+                <legend>Select a colour</legend>
+                <input id="karmaorange" type="radio" name="colour" value="karmaorange" />
+                <label for="karmaorange">Karma Orange</label>
+                
+                
+                    
+                
+                <div class="spacegrey">
+                    <input id="spacegrey" type="radio" name="colour" value="spacegrey" />
+                    <label for="spacegrey">Space Grey</label>
+                    
+                    
+                    
+                </div>
+            </div>
+            </fieldset>
+<!--agrement-->
+            <fieldset class="agrement">
+                <input id="agrement" type="checkbox" name="agrement" />
+                <label for="agrement">By placing your order you agree to Karma's<a href="#"> Terms and Conditions.</a>
+                *</label>
+                    
+                
+
+            </fieldset>
+                    <button type="Place my order">Place my order</button>
+                    
+         </form>
+         </section>
+<!--img-->
+        <section class="image-container">
+            <img src="img/Girl working at desk.png" alt="Girl using Karma in a cafe" />
+    
+            </section>
+        </section>
+    </main>
+    
+
+  
+  
+  
+  
+<!--footer-->
+  <footer>
+    <p>Join us on</p>
+    <div class="social-icon">
+
+        <a href="twitter"></a>
+        <img src="img/twitter-icon.svg" alt class="icon">
+        </img>
+        <a href="facebook"></a>
+        <img src="img/facebook-icon.svg" alt class="icon">
+        </img>
+        <a href="instagram"></a>
+        <img src="img/instagram-icon.svg" alt class="icon">
+        </img>
+
+    </div>
```

Comment:
Change the social icons section to put the images inside a tags, so that you can style them as in the reference picture:
```suggestion
<div class="social-container">
            <a href="#">
                <img src="./img/twitter-icon.svg" alt="Find us on Twitter" class="social-logo" />
            </a>
            <a href="#">
                <img src="./img/facebook-icon.svg" alt="Find us on Facebook" class="social-logo" />
            </a>
            <a href="#">
                <img src="./img/instagram-icon.svg" alt="Find us on Instagram" class="social-logo" />
            </a>
        </div>
```